packages/storefront-lwc/src/modules/commerce/data/basket.js

```diff
@@ -129,29 +140,7 @@ class Basket {
                 mutation: gql`
                     mutation {
                         updateShippingMethod(basketId: "${basketId}", shipmentId: "${shipmentId}", shippingMethodId: "${shippingMethodId}") {
-                            basketId
-                            customerId
-                            getBasketMessage
-                            totalProductsQuantity
-                            shipmentId
-                            shipmentTotal
-                            selectedShippingMethodId
-                            products {
-                                productId
-                                itemId
-                                quantity
-                                productName
-                                price
-                            }
-                            orderTotal
-                            orderLevelPriceAdjustment {
-                                itemText
-                                price
-                            }
-                            shippingTotal
-                            shippingTotalTax
-                            taxation
-                            taxTotal
+                            ${this.getBasketAttributes}
```

Comment:
re using the const above and asking for more fields that will be useful once we include rest of apollo client stuff